### DISCUSSION TITLE
v2ray -test 测试没问题，IP端口也对应，没有报错，就是无法连接，感觉是客户端程序没有连接到服务器

### DISCUSSION THREAD

**Discussion by shuidaan** [v2fly/v2ray-core#624] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/24624971/105439650-32b24a00-5ca0-11eb-9fa7-848f53a34783.png)
![image](https://user-images.githubusercontent.com/24624971/105439655-35ad3a80-5ca0-11eb-9506-aad6744227e0.png)

为什么服务器哪里一点数据都收不到，感觉好像没有连接似的，核对过服务器IP port 都对应的。

![image](https://user-images.githubusercontent.com/24624971/105439910-a5bbc080-5ca0-11eb-9d49-37f287951fee.png)
服务器状态正常 端口也是开放的。测试过端口，正常。

**Comment by shuidaan** [comment, marked as answer]
1.5.2. 客户端提示 Socks: unknown Socks version:
可能原因：客户端配置的 inboud 设置成了 socks 而浏览器的代理协议设置为 http。

修正方法：修改配置文件使客户端的 inboud 的 protocol 和浏览器代理设置的协议保持一致。
https://toutyrater.github.io/basic/vmess.html

**Comment by CalmLong** [comment]
~~自己帮助自己~~
